### ISSUE TITLE
Seeking comments on Code of Conduct 

### ISSUE THREAD

**Issue by chiaral** [pangeo-data/governance#12]
The code of conduct draft is located here:
https://github.com/pangeo-data/governance/blob/master/conduct/code_of_conduct.md

It was, similarly to the rest of the governance document so far, drafted based on on Project Jupyter's governance repo: 
https://github.com/jupyter/governance/blob/master/conduct/code_of_conduct.md

Similarly to the governance document (#10) I am inviting anyone to comment on this document, for the next couple of weeks. 

You can comment within this issue, or [open a new issue ](https://github.com/pangeo-data/governance/issues/new) if you feel that commenting within this thread is not the appropriate place.

**Comment by mcgibbon** [comment]
Those look like great places to take pointers for our Code of Conduct. Some things I notice right away:

- The current code of conduct is a relatively big wall of text. The bolded bullet points further down help with quickly retaining the information, but it's only after several large paragraphs. I appreciate the more concise style of all four examples you linked. There should be a very low barrier to absorbing the most important information in this document.
- Describing consequences and reporting procedures is important. Our information about consequences is also much more verbose than the examples you linked. I appreciate those communities giving themselves broad permission to take whatever action they deem appropriate in the case of harassment. While that process likely involves the same sorts of things in our [enforcement manual](https://github.com/pangeo-data/governance/blob/master/conduct/enforcement.md), I feel more confident about a harassment report being followed upon to a satisfactory resolution in the terse, upfront way consequences are described in the other documents. Perhaps following up that information with a more descriptive procedural document would be appropriate, but it does not replace the need for more straightforward wording.
- The documents you linked have sections which explicitly define harassment and discrimination. This seems like a good idea.

Thank you for taking it on yourself to put together a PR taking from these documents @chiaral! I wish I had more time to commit to my own ideas on this.

**Comment by chiaral** [comment]
Yes, I will. It might take me a few days, but I will work on it.

**Comment by kmpaul** [comment]
Just curious how this is going.

**Comment by chiaral** [comment]
Hi! I actually started working on a couple of options today. I will submit a PR later today 👍

**Comment by kmpaul** [comment]
Oooh!  Great timing.  :-)

**Comment by kmpaul** [comment]
I think this PR (#16) was submitted, approved, and merged.  Should we close it?
